k8s/namespaces/wa/wa-workflow-api/wa-workflow-api.yaml

```diff
@@ -13,3 +13,4 @@ spec:
       replicas: 2
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/wa-workflow-api/service:prod-8431c386
+      ingressHost: wa-workflow-api.service.core-compute-aat.internal
```

Comment:
think should be something like: 
`ingressHost: wa-workflow-api-aat.service.core-compute-aat.internal`
e.g. ia-case-api-aat.service.core-compute-aat.internal/